code/modules/urist/_maps/templates.dm

```diff
@@ -216,3 +215,7 @@ var/list/datum/map_template/underground_templates = list()
 
 	qdel(src)
 
+
+/obj/effect/template_loader/gamemode/assault
+	var/maptype = 0
+	gamemode = "assault"
```

Comment:
It might be better in the long run to make it into a macro, i.e. define MODE_ASSAULT 1 or whatever, for those if you want them to be dynamically loaded for each gamemode.

It'll be equally readable, prevent issues due to misspells, and will be significantly faster when you actually want to check the value of gamemode the loader holds, since string comparisons are slow - and if there's multiple loaders to iterate on, this WILL matter.